pkg/kapp/clusterapply/delete_change.go

```diff
@@ -122,3 +158,16 @@ func descMessage(res ctlres.Resource) []string {
 	}
 	return []string{}
 }
+
+func (c DeleteChange) isInoperableResource() bool {
+
+	res := c.change.ExistingResource()
+
+	for _, r := range inoperableResourceList {
+		strings.EqualFold(r.Name, res.Name())
+		if strings.EqualFold(r.Name, res.Name()) && strings.EqualFold(r.Kind, res.GroupKind().Kind) && strings.EqualFold(r.Group, res.GroupKind().Group) {
```

Comment:
As renu pointed out, I do not think we should do a match which ignores cases here if we have decided on using Group-Kinds to match as they are always the same.